src/redux/books/Books.js

```diff
@@ -1,9 +1,35 @@
 import React, { useState } from 'react';
 import { v4 as uuidv4 } from 'uuid';
-import BookList from '../components/BookList';
-import AddBook from '../components/AddBook';
+import BookList from '../../components/BookList';
+import AddBook from '../../components/AddBook';
 
-const Books = () => {
+const ADD_BOOK = 'bookstore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
+
+export function addBook(book) {
+  return { type: ADD_BOOK, book };
+}
+
+export function removeBook(book) {
+  return { type: REMOVE_BOOK, book };
+}
+
```

Comment:
- [ ] Actually, I can create a book from the form but the remove button is no longer working, Kindly fix this 💯 👍🏻

Reply:
Hi @BenMukebo, 

Thanks for your feedback. There is no requirement for this. Will you please check again the requirements.

Thanks again